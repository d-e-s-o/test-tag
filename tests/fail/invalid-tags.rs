// Copyright (C) 2024 Daniel Mueller <deso@posteo.net>
// SPDX-License-Identifier: (Apache-2.0 OR MIT)

/// Invalid specifiers in tag list.
#[test_tag::tag(abc!@*()!@)]
#[test]
fn cant_compile() {}
