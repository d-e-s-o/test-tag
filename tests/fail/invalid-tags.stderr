error: expected `,`
 --> tests/fail/invalid-tags.rs:5:20
  |
5 | #[test_tag::tag(abc!@*()!@)]
  |                    ^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/fail/invalid-tags.rs:7:21
  |
7 | fn cant_compile() {}
  |                     ^ consider adding a `main` function to `$DIR/tests/fail/invalid-tags.rs`
